jtec-instrumentation/src/main/java/edu/tum/sse/jtec/util/IOUtils.java

```diff
@@ -71,8 +72,14 @@ public static void writeToFile(final Path file, final String message, final bool
         }
     }
 
+    public static String readFromFile(final Path file, final Charset encoding) throws IOException
+    {
+        byte[] encoded = Files.readAllBytes(file);
+        return new String(encoded, encoding);
+    }
+
     public static String readFromFile(final Path file) throws IOException {
-        return new String(Files.readAllBytes(file));
+        return readFromFile(file, StandardCharsets.UTF_8);
```

Comment:
What do you think about extracting a global charset for JTeC and using it for all reading and writing? That could help with avoiding inconsistencies.